Makefile

```diff
@@ -229,25 +229,24 @@ tool: $(TOOLS_BIN)
 include mk/riscv-arch-test.mk
 include mk/tests.mk
 
-CHECK_ELF_FILES := \
-	hello \
-	puzzle \
-	fcalc
+# the prebuilt executables are built for "rv32im"
```

Comment:
If rv32emu is configured as RV32I-only, the target `make check` should validate `hello.elf`.